src/main/resources/com/google/api/codegen/py/setup.py.snip

```diff
@@ -18,6 +18,9 @@
       @end
   ]
 
+  with io.open('README.rst', 'r', encoding='utf-8'): as readme_file:
```

Comment:
This is a `SyntaxError` (note the two colons)

Reply:
Done